inst/examples/translation.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
(1, 2) I came up with jsons, as I thought it would be a good for having configuration and translations in one file. Good for clarity and convenient for programmers, but you are right that might be laborious for translators. As you are second person raising this concern I switch to CSV solution with separate files for languages.

(3) Cool, I will leave that JSON as an option, but CSV approach will be set as a default .